jenkins/deduper.py

```diff
@@ -41,8 +41,8 @@ def _aborted_description(current_build_id, pr):
 
         :Returns: A description (string)
         """
-        return ("[PR #{}] Build automatically aborted because"
-                " there is a newer build for the same PR. See build"
+        return ("[PR #{}] You have a newer build running for this PR."
+                " Go see its results instead! Build"
```

Comment:
Also I added a FAQ about this, so as the language is changing be sure to update https://openedx.atlassian.net/wiki/display/TE/Jenkins+Guide#JenkinsGuide-Itriedtouse"jenkinsrunxxxx"butIstillseearedX,whatgives?